k8s/namespaces/rd/rd-professional-api/perftest.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   releaseName: rd-professional-api
   values:
     java:
-      image: hmctspublic.azurecr.io/rd/professional-api:prod-8e5d233-20220706160335 #{"$imagepolicy": "flux-system:rd-professional-api"}
+      image: hmctspublic.azurecr.io/rd/professional-api:pr-1292-1189787-20220713103900 #{"$imagepolicy": "flux-system:perftest-rd-professional-api"}
```

Comment:
I'm not sure the perftest value needs adding here, can you double check?

Reply:
yes, we need as this being referred in this image policy - https://github.com/hmcts/cnp-flux-config/blob/master/apps/rd/rd-professional-api/perftest-image-policy.yaml#L4